.github/workflows/autofix.yml

```diff
@@ -0,0 +1,27 @@
+name: autofix.ci # needed to securely identify the workflow
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Does autofix add extra commit? If so, it should not apply to main branch.

Reply:
Let me experiment